test/bundleUtilConsts.js

```diff
@@ -0,0 +1,121 @@
+const URN_REPLACE_REFERENCES_ENTRIES = [
+  {
+    fullUrl: 'urn:uuid:61ebe359-bfdc-4613-8bf2-c5e300945f0a',
+    resource: {
+      resourceType: 'Patient',
+      gender: 'male',
+      birthDate: '1974-12-25'
+    },
+    request: {
+      method: 'POST',
+      url: 'Patient'
+    }
+  },
+  {
+    fullUrl: 'urn:uuid:88f151c0-a954-468a-88bd-5ae15c08e059',
+    resource: {
+      resourceType: 'Patient',
+      gender: 'male',
+      birthDate: '1974-12-25',
+      reference: 'urn:uuid:61ebe359-bfdc-4613-8bf2-c5e300945f0a'
+    },
+    request: {
+      method: 'POST',
+      url: 'Patient',
+      ifNoneExist: 'identifier=http:/example.org/fhir/ids|234234'
```

Comment:
huh? This URL looks a bit wrong. I don't believe we support this attribute in bundle posting. This could also be removed.
```suggestion
      ifNoneExist: 'identifier=http://example.org/fhir/ids|234234'
```